src/apps/Console/console-utils.test.js

```diff
@@ -103,4 +132,14 @@ describe('parsePermissions tests', () => {
       ['VOTE_ROLE', '0x', '0x'],
     ])
   })
+
+  test('Errors out on wrong types', () => {
```

Comment:
Generally I find it a bit nicer to leave the error case (and more "exotic") tests towards the end of the suite. Having the most basic functionality tested first makes it easier to ascertain a unit's general behaviour from a quick glance.